controllers/categories.controllers.js

```diff
@@ -20,3 +21,13 @@ exports.getEndpoints = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+
+  fetchReviewId(review_id)
+    .then((reviewId) => {
```

Comment:
It would be better to send the review object back (instead of the whole array) on a key of something descriptive reviewId. Simply "review" would be good as it makes more sense client-side. 

Ideally change the test first to expect the change and then update the functionality to pass the test